src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$cellsize: 100px;
+$gap: 1px;
+$days: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.calendar-wrapper {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: $cellsize * 7 + $gap * 6;
```

Comment:
The calculation for the width of the .calendar class should be enclosed in parentheses to ensure proper arithmetic operation order. It should be `($cellsize * 7) + ($gap * 6);` to avoid any potential issues with operator precedence.